src/components/shared/Dropdown.css

```diff
@@ -48,6 +48,30 @@ html[dir="rtl"] .dropdown {
   background-color: var(--search-overlays-semitransparent);
 }
 
+.dropdown li img.prettyPrint {
+  -webkit-mask: url(/images/prettyPrint.svg) no-repeat;
+  mask: url(/images/prettyPrint.svg) no-repeat;
+  -webkit-mask-size: 100%;
+  mask-size: 100%;
+  background: var(--theme-highlight-blue);
+  width: 13px;
```

Comment:
this looks like a magic number, maybe we can make it a variable and name it?

Reply:
Exported to variables.css as `--icon-size`